src/outputs/outputs_main.py

```diff
@@ -170,6 +171,20 @@ def run_outputs(
         )
         OutputMainLogger.info("Finished  Intram by ITL1 output.")
 
+    # Running Intram UK by ITL 1 and 2
+    if config["global"]["output_intram_uk_itl2"]:
+        OutputMainLogger.info("Starting  Intram UK by ITL 1 and 2 output...")
+        output_intram_uk_itl_1_2(
+            outputs_df,
+            ni_full_responses,
+            config,
+            write_csv,
+            run_id,
+            postcode_mapper,
+            itl_mapper,
+        )
+        OutputMainLogger.info("Finished  Intram UK by ITL 1 and 2 output.")
```

Comment:
In the ITL1 output TLF is appearing twice (once as 'East Midlands' and once as  'East Midlands (England)').

TLH is also showing as just 'East' - is that right?

Reply:
'East Midlands' and once as 'East Midlands (England)' - was an error in my mapper, fixed in Windows, needs fixing in DAP.

TLH Is 'East', it's OK

Reply:
Updated the mapper in DAP